templates/_layouts/examples.html

```diff
@@ -49,7 +49,7 @@
     </script>
   </head>
 
-  <body>
+  <body class="{% block bodyClass %}{% endblock %}">
```

Comment:
Is this still needed? Feels like it might be leftover from the other proposed way of implementing the sticky footer.

Reply:
Ah, good catch